internal/rbd/driver/driver.go

```diff
@@ -213,7 +211,9 @@ func (r *Driver) setupCSIAddonsServer(conf *util.Config) error {
 	r.cas.RegisterService(is)
 
 	if conf.IsControllerServer {
-		rs := casrbd.NewReclaimSpaceControllerServer()
+		rs := casrbd.NewReclaimSpaceControllerServer(&rbd.ControllerServer{
+			VolumeLocks: util.NewVolumeLocks(),
+		})
```

Comment:
I dont think we need to define locking here, we already have `casrbd.NewReplicationServer(NewControllerServer(r.cd))` at line 222 which is already initializing the locking internally we don't need to do it explicitly.